test/builtins/Transformers.jl

```diff
@@ -465,6 +465,57 @@ end
     @test infos[:name] == "OneHotEncoder"
     @test infos[:input_scitype] == MLJBase.Table
     @test infos[:output_scitype] == MLJBase.Table
+
+    # test the work on missing values
+    X = (name   = categorical(["Ben", "John", "Mary", "John", missing], ordered=true),
+         height = [1.85, 1.67, 1.5, 1.67],
+         favourite_number = categorical([7, 5, 10, missing, 5]),
+         age    = [23, 23, 14, 23])
+
+    t  = OneHotEncoder()
+    f, _, report = @test_logs((:info, r"Spawning 3"),
+                         (:info, r"Spawning 3"), MLJBase.fit(t, 1, X))
+     
+    Xt = MLJBase.transform(t, f, X)
+
+    @test typeof(Xt.name__John == float.([false, true, false, true, missing])) == Missing
```

Comment:
Line 481 seems like a rather weak test. I think we should check all the elements have the expected form. How about

```julia
@test length(Xt.name__John) == 5
@test skipmissing(Xt.name__John) == float.([false, true, false, true])
@test ismissing(XT.name__John[5] )
```
?

Similarly for line 483, 505, and 507.

Reply:
Oh, this is nice. I'm very sure that I don't know many of the functions related to missing 🙂